src/nimvscode/jsNode.nim

```diff
@@ -34,14 +37,22 @@ proc newBuffer*(size:cint):Buffer {.importcpp: "(new Buffer(@))".}
     ## TODO - mark as deprecated
 proc bufferAlloc*(size:cint):Buffer {.importcpp: "(Buffer.alloc(@))".}
 
+proc newArray*[T](size=0):Array[T] {.importcpp: "(new Array(@))".}
+
 # global
 proc setInterval*(g:GlobalModule, f:proc():void, t:cint):void {.importcpp.}
 
+# Array
+proc `[]`*[T](a:Array[T]):T {.importcpp: "#[#]".}
+proc `[]=`*[T](a:Array[T],val:T):T {.importcpp: "#[#]=#".}
+proc push*[T](a:Array[T],val:T) {.importcpp: "#.push(#)".}
+
 # Map
 proc get*[K,V](m:Map[K,V], key:K):V {.importcpp.}
 proc set*[K,V](m:Map[K,V], key:K, value:V):void {.importcpp.}
```

Comment:
```suggestion
proc `[]`*[K,V](m:Map[K,V], key:K):V {.importcpp:"#.get(key)".}
proc `[]=`*[K,V](m:Map[K,V], key:K, value:V):void {.importcpp:"#.set(key, value)".}
proc get*[K,V](m:Map[K,V], key:K):V {.importcpp, deprecated.}
proc set*[K,V](m:Map[K,V], key:K, value:V):void {.importcpp, deprecated.}
```

Reply:
There are a few warnings in `elrpc.nim`, but those can be take care of later.

Down below I would add a copy of `entries` renamed `pairs` and mark `entries` with  `{.deprecated.}`.